src/Extension.php

```diff
@@ -88,6 +100,9 @@ public function getConfig(ArrayNodeDefinition $builder)
                         scalarNode('dbprefix')->
                             defaultValue(null)->
                         end()->
+                        arrayNode('connection_options')->
+                            ignoreExtraKeys(false)->
```

Comment:
Ignore me, i've checked this doesn't break the configuration at all. But does need some tests, will attempt to add them for you :)